Develop

ColinT152 opened the pull request.

RobPavey commented:
This PR is changing 2 files that it should not - looks like merge errors:
extension/site/np/core/np_build_citation.mjs
extension/site/np/core/np_extract_data.mjs

RobPavey commented on unit_tests/bg/citations/ref/francis_weyland_dunn_d_1874.json:
```diff
@@ -1,14 +1,14 @@
 {
   "std_inline": {
-    "citation": "<ref>\n'''Memorial''':\n\"Billion Graves\"<br/>\nOak Grove Cemetery, 2-12 W Montgomery St, Hillsdale, Michigan, United States<br/>\n[https://billiongraves.com/grave/Francis-Wayland-Dunn/7557769 BillionGraves memorial] (accessed 6 May 2021)<br/>\nMemorial page for Francis Wayland Dunn (29 January 1843-13 December 1874); Transcribed by Tellinghistory, April 15, 2014; Photographed b ...
```
Is this correct? It seems like it is losing the state name?

RobPavey commented on unit_tests/bg/citations/ref/margaret_ann_craney_d_1909.json:
```diff
@@ -1,14 +1,14 @@
 {
   "std_inline": {
-    "citation": "<ref>\n'''Memorial''':\n\"Billion Graves\"<br/>\nNewcastle upon Tyne (Saint Nicholas) Cemetery, Westgate Road, Newcastle upon Tyne, England, United Kingdom<br/>\n[https://billiongraves.com/grave/Margaret-Ann-Craney/27550409 BillionGraves memorial] (accessed 6 May 2021)<br/>\nMemorial page for Margaret Ann Craney (d August 1909); Transcribed by DdraigGoch, June 1, 2018; Photograph ...
+    "citation": "<ref>\n'''Memorial''':\n\"Billion Graves\"<br/>\nNewcastle upon Tyne (Saint Nicholas) Cemetery, Westgate Road, Newcastle upon Tyne, Tyne and Wear, United Kingdom<br/>\n[https://billiongraves.com/grave/Margaret-Ann-Craney/27550409 BillionGraves memorial] (accessed 6 May 2021)<br/>\nMemorial page for Margaret Ann Craney (d August 1909); Transcribed by DdraigGoch, June 1, 2018; Phot ...
```
It seems like it is replacing the country with the county. Is that intentional?

ColinT152 replied:
Fixed with latest update

RobPavey commented on unit_tests/bg/search_url/ref/thomas_moffat_29_october_new_style.json:
```diff
@@ -1,5 +1,5 @@
 {
   "std": {
-    "url": "https://billiongraves.com/search/results?given_names=Thomas&family_names=Moffat&cemetery_state=Scotland&cemetery_country=United%20Kingdom&year_range=5&size=15"
+    "url": "https://billiongraves.com/search/results?CollectionID=&CatalogID=&PageNumber=1&PageSize=20&EventBirthYearRange=5&EventDeathYearRange=5"
```
Seems like this has less information than before?

ColinT152 replied:
Due to failure to gather cemetery data. Now fixed.

RobPavey reviewed (commented):
A few comments from a quick review. I will sync latest and try it out.

ColinT152 commented:
Fixed the cemetery address issues. The search issue is a consequence of the
problems I was having with extracting the cemetery info from the new style
site. Now fixed.

Have uploaded my develop branch just now.


On Fri, 28 Oct 2022 at 18:45, RobPavey ***@***.***> wrote:

> ***@***.**** commented on this pull request.
>
> A few comments from a quick review. I will sync latest and try it out.
> ------------------------------
>
> In unit_tests/bg/citations/ref/francis_weyland_dunn_d_1874.json
> <https://github.com/RobPavey/wikitree-sourcer/pull/6#discussion_r1008305937>
> :
>
> > @@ -1,14 +1,14 @@
>  {
>    "std_inline": {
> -    "citation": "<ref>\n'''Memorial''':\n\"Billion Graves\"<br/>\nOak Grove Cemetery, 2-12 W Montgomery St, Hillsdale, Michigan, United States<br/>\n[https://billiongraves.com/grave/Francis-Wayland-Dunn/7557769 BillionGraves memorial] (accessed 6 May 2021)<br/>\nMemorial page for Francis Wayland Dunn (29 January 1843-13 December 1874); Transcribed by Tellinghistory, April 15, 2014; Photographed by Tellinghistory, April 15, 2014.\n</ref>",
>
> Is this correct? It seems like it is losing the state name?
> ------------------------------
>
> In unit_tests/bg/citations/ref/margaret_ann_craney_d_1909.json
> <https://github.com/RobPavey/wikitree-sourcer/pull/6#discussion_r1008306450>
> :
>
> > @@ -1,14 +1,14 @@
>  {
>    "std_inline": {
> -    "citation": "<ref>\n'''Memorial''':\n\"Billion Graves\"<br/>\nNewcastle upon Tyne (Saint Nicholas) Cemetery, Westgate Road, Newcastle upon Tyne, England, United Kingdom<br/>\n[https://billiongraves.com/grave/Margaret-Ann-Craney/27550409 BillionGraves memorial] (accessed 6 May 2021)<br/>\nMemorial page for Margaret Ann Craney (d August 1909); Transcribed by DdraigGoch, June 1, 2018; Photographed by Lindsay, May 25, 2018.<br/>Also on memorial : Hugh Craney d 1951.\n</ref>",
> +    "citation": "<ref>\n'''Memorial''':\n\"Billion Graves\"<br/>\nNewcastle upon Tyne (Saint Nicholas) Cemetery, Westgate Road, Newcastle upon Tyne, Tyne and Wear, United Kingdom<br/>\n[https://billiongraves.com/grave/Margaret-Ann-Craney/27550409 BillionGraves memorial] (accessed 6 May 2021)<br/>\nMemorial page for Margaret Ann Craney (d August 1909); Transcribed by DdraigGoch, June 1, 2018; Photographed by Lindsay, May 25, 2018.<br/>Also on memorial : Hugh Craney d 1951.\n</ref>",
>
> It seems like it is replacing the country with the county. Is that
> intentional?
> ------------------------------
>
> In unit_tests/bg/search_url/ref/thomas_moffat_29_october_new_style.json
> <https://github.com/RobPavey/wikitree-sourcer/pull/6#discussion_r1008307565>
> :
>
> > @@ -1,5 +1,5 @@
>  {
>    "std": {
> -    "url": "https://billiongraves.com/search/results?given_names=Thomas&family_names=Moffat&cemetery_state=Scotland&cemetery_country=United%20Kingdom&year_range=5&size=15"
> +    "url": "https://billiongraves.com/search/results?CollectionID=&CatalogID=&PageNumber=1&PageSize=20&EventBirthYearRange=5&EventDeathYearRange=5"
>
> Seems like this has less information than before?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/RobPavey/wikitree-sourcer/pull/6#pullrequestreview-1160417676>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AEVL22VLZTKOBOFOWOIA7O3WFQGM5ANCNFSM6AAAAAARPNUDSE>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
